sdk/documenttranslation/azure-ai-documenttranslation/azure/ai/documenttranslation/_polling.py

```diff
@@ -0,0 +1,64 @@
+#my_polling 
+
+from azure.core.polling.base_polling import (
+    LongRunningOperation,
+    _is_empty,
+    _as_json,
+    BadResponse,
+    OperationFailed
+)
+
+
+class TranslationPolling(LongRunningOperation):
```

Comment:
This impl looks good to me. Since we are inheriting from the LongRunningOperation abstract class, we just need to define a `get_final_get_url` method. Since the operation doesn't need to do a final get, we can just return None like this:

```
    def get_final_get_url(self, pipeline_response):
        return None
```

Reply:
yeah sorry, forgot it's an abstract method!